### PR TITLE
bump to 3.0.4

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: shirly <AndreMouche@126.com>

###  What have you changed?

Bump to 3.0.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [3.0.4]
+
++ Fix the issue that the approximate keys is not correct when region is empty [#5403](https://github.com/tikv/tikv/pull/5403).
```
Comment (zhangjinpeng1987) [inline CHANGELOG.md]: Seems this pr is merged in master branch?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
++ Make the config support rocksdb doubly skiplist to optimize `reverse-scan` [#5368](https://github.com/tikv/tikv/pull/5368).
++ Optimize point-get in coprocessor [#5463](https://github.com/tikv/tikv/pull/5463).
++ Support batch-split command and empty batch command [#5470](https://github.com/tikv/tikv/pull/5470).
++ Fix `PointGetter` performance issue when there are concurrent write [#5495](https://github.com/tikv/tikv/pull/5495).
++ Fix the output on short version flag [#5501](https://github.com/tikv/tikv/pull/5501).
++ Support the pessmistic transaction API: txn-heart-beat [#5507](https://github.com/tikv/tikv/pull/5507).
++ `titan` GC and monitoring improvement [#5517](https://github.com/tikv/tikv/pull/5517).
++ Update `grpcio` to v0.4.5 [#5523](https://github.com/tikv/tikv/pull/5523).
++ Support GRPC memory quota [#5524](https://github.com/tikv/tikv/pull/5524).
++ Fix commit index is not forwarded when merge entry is empty [#5526](https://github.com/tikv/tikv/pull/5526).
```
Comment (disksing) [inline CHANGELOG.md]: delete the `.` at end of line?

**Pull request merged**